server/configureChangeFeeds.js

```diff
@@ -26,7 +26,12 @@ function _getQueue(queueName) {
 
 function _getFeed() {
   return r.table('users')
-    .getAll(USER_ROLES.LEARNER, {index: 'roles'})
+    .filter(filteredUser => {
+      const updatedAt = filteredUser('updatedAt')
+      const isALearner = filteredUser('roles').contains(USER_ROLES.LEARNER)
+      const isNewlyCreated = filteredUser('createdAt').during(updatedAt.sub(10), updatedAt.add(10))
+      return isALearner && isNewlyCreated
```

Comment:
Clever solution! But let's try to get to the bottom of why `.filter(r.row('old_val').eq(null))` wasn't working as expected and focus our efforts there. We can't reliably count on equal timestamps for `createdAt` and `updatedAt` being a true indicator of a new insert.

In our live review today, we discovered that the use of `getAll()` in the changefeed query has an effect on whether or not `old_val` is null for a change after an update. Since we don't actually need to filter by the `LEARNER` role, let's just get rid of that altogether. That will resolve the changefeed filter problem and also allow us to avoid the extra logic for comparing timestamps.

Issue for updating the filter tracked here: https://github.com/LearnersGuild/idm/issues/203